code/game/machinery/iv_drip.dm

```diff
@@ -170,12 +147,16 @@
 	if(!container_reagents)
 		return
 
+	//The thresholds used to determine the reagent fill icon
+	var/static/list/fill_icon_thresholds = list(0, 10, 25, 50, 75, 80, 90)
+
 	var/threshold = null
 	for(var/i in 1 to fill_icon_thresholds.len)
 		if(ROUND_UP(100 * container_reagents.total_volume / container_reagents.maximum_volume) >= fill_icon_thresholds[i])
 			threshold = i
+
 	if(threshold)
-		var/fill_name = "[fill_icon_state][fill_icon_thresholds[threshold]]"
+		var/fill_name = "reagent[fill_icon_thresholds[threshold]]"
```

Comment:
why do we want to remove support for ``fill_icon_state``?

Reply:
Because nobody changed thi var value. It had no references anywhere so might as well make it a constant